src/App.tsx

```diff
@@ -1,56 +1,73 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import DropdownMenu from './components/DropdownMenu/DropdownMenu';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [data] = useState<Person[]>(peopleFromServer);
```

Comment:
for what we need `useState` here? we don't change data, i think it's redundant in this case